### DISCUSSION TITLE
Enunciado QMP uso de listas

### DISCUSSION THREAD

**Discussion by tobiascastaneda2000** [dds-jv/foro#185] [Consultas generales] [answered]
Buenas Noches!! Tengo un problema para entender lo que me pide. No se si es esa justamente la idea del ejercicio por lo que no debería preguntar eso jeje, pero por las dudas lo hago.

El codigo lo tengo masomenos armado, pero tengo muchas dudas de como platear el tema de los atuendos:

Yo tenia pensado en crear una clase llamada usuario, en la que le daria como atributo atuendo, que seria una lista de Prendas que se pueden ir agregando gracias a un metodo, en caso de que la validacion de la misma este bien.

Esta bien planteado asi? Por que la clase usuario solamente seria usada para eso, y quizas hay otra forma de armar el codigo de forma mas sencilla.

Disculpe las molesyias, espero su respuesta.

**Comment by flbulgarelli** [comment, marked as answer]
¡Hola!

Es un enfoque posible, pero en clase y a lo largo de las primeras iteraciones iremos discutiendo sus pros y contras. 

¡Saludos!
